server/.env

```diff
@@ -0,0 +1,3 @@
+GOOGLE_CIVIC_API_KEY=AIzaSyDjSjXjosp_JcNvXWEjTgwYEHeHs9o2y8U
```

Comment:
I'm guessing this key needs to be removed (keys are not generally public info and someone like Michelle can store it!). If this is a public key that is not associated with you, an email, etc., then it would be fine.

Reply:
Then this file would be udpated to 
`GOOGLE_CIVIC_API_KEY=`
And the readme would have info about how to fill it in